index.bs

```diff
@@ -1616,7 +1612,7 @@ To <dfn for="WebTransportSendStream">write</dfn> |chunk| to a {{WebTransportSend
      except for [=flow control=] reasons or [=WritableStream/Error | error=].
 
      If the sending of |bytes| cannot be completed in its entirety within the current
-     [=flow control=] window, and |atomic| is true, then give up on sending |bytes| and
+     [=flow control=] window, and |atomic| is true, then stop sending |bytes| and
```

Comment:
Maybe the right way to deal with this problem is to have three distinct outcomes from this step, so that you can execute the correct code on subsequent steps.